spec/spec_helper.rb

```diff
@@ -5,13 +5,16 @@
 
 Dir[Rails.root.join("spec/shared/**/*.rb")].sort.each { |f| require f }
 Dir[File.join(__dir__, "support/**/*.rb")].sort.each { |f| require f }
+Dir[ManageIQ::Providers::Openstack::Engine.root.join("spec/support/**/*.rb")].each { |f| require f }
```

Comment:
As per rubocop, this is generally safer:

```suggestion
Dir[ManageIQ::Providers::Openstack::Engine.root.join("spec/support/**/*.rb")].sort.each { |f| require f }
```